src/main/java/de/retest/recheck/auth/RetestAuthentication.java

```diff
@@ -177,13 +181,16 @@ private boolean isTokenValid() {
 	}
 
 	static KeycloakResult getRequestParameters( final String request ) {
-		final HttpUrl url = HttpUrl.parse( "http://localhost/" ).resolve( request.split( " " )[1] );
+		final String url = "http://localhost/" + request.split( " " )[1];
+		final Map<String, String> parameters = URLEncodedUtils.parse( URI.create( url ), Charset.forName( "UTF-8" ) )//
+				.stream()//
```

Comment:
Sometimes you do `...)//` (e.g. here and line 193) and sometimes `...) //`.

Reply:
:see_no_evil: